routers/repo/pull.go

```diff
@@ -182,11 +182,17 @@ func checkPullInfo(ctx *context.Context) *models.Issue {
 // PrepareMergedViewPullInfo show meta information for a merged pull request view page
 func PrepareMergedViewPullInfo(ctx *context.Context, issue *models.Issue) {
 	pull := issue.PullRequest
-	ctx.Data["HasMerged"] = true
-	ctx.Data["HeadTarget"] = issue.PullRequest.HeadUserName + "/" + pull.HeadBranch
-	ctx.Data["BaseTarget"] = ctx.Repo.Owner.Name + "/" + pull.BaseBranch
 
 	var err error
+	if err = pull.GetHeadRepo(); err != nil {
```

Comment:
If PR is made and than forked repository is deleted there will be error `ErrRepoNotExist` and it will not be possible to open PR even to close it. It should at least fall back to message `... from fork_owner:branch into branch` format in such case.
Currently in such case this codes gets `PANIC: runtime error: invalid memory address or nil pointer dereference`

Reply:
Probably I could be wrong that it returns error.., More looks like it just sets `pull.HeadRepo` as `nil`